libnm-core/nm-setting-wpan.c

```diff
@@ -270,5 +357,33 @@ nm_setting_wpan_class_init (NMSettingWpanClass *klass)
 	                            G_PARAM_READWRITE |
 	                            G_PARAM_STATIC_STRINGS));
 
+	/**
+	 * NMSettingWpan:page:
+	 *
+	 * IEEE 802.15.4 channel page.
+	 **/
+	g_object_class_install_property
+	        (object_class, PROP_PAGE,
+	         g_param_spec_int (NM_SETTING_WPAN_PAGE, "", "",
+	                            NM_SETTING_WPAN_PAGE_DEFAULT,
+		                    IEEE802154_MAX_PAGE,
+		                    NM_SETTING_WPAN_PAGE_DEFAULT,
+	                            G_PARAM_READWRITE |
+	                            G_PARAM_STATIC_STRINGS));
```

Comment:
indentation

Reply:
If you restrict the range here, you don't need to `verify()` it. But restricting the range in the GOjbect property is generally not a good idea, because it means the range cannot be extended. It may be better to assume a range of `G_MININT32 .. G_MAXINT32` and validate it in `verify()`.

Also, `verify()` validates that the value is not larger than MAX, but it does not validate that the value is smaller than `-1`. It should (provided, you verify the range there).

Reply:
Fixed; along with the checkpatch.pl changes to detect the whitespace sins.